app/models/post.rb

```diff
@@ -0,0 +1,3 @@
+class Post < ActiveRecord::Base
+	has_many :comments, dependent: :destroy
+end
```

Comment:
You could have included validations on the model to help ensure that the data is what you want before it is saved.
https://github.com/ga-wdi-lessons/rails-errors-validations-flash/blob/master/validations.md#declare-validations-on-your-models